Egregious-Eggnog/reactive-nutshell
# Sh articles modals test

# Description

Refined Login

Fixes # (issue)

## Type of change

Made a lot of changes - please check all functionality and database info to make sure that things are posting as expected


# Testing Instructions

git fetch --all
git checkout sh-articles-modals-test
npm start
json-server -p 5002 -w database.json

check everything with regards to login - do all pages redirect to login when you're not logged in
check logout and inability to access information
navbar should only show links when you're logged in
log in link/page only when you're logged out
check all functionality (CRUD) for your individual components


# Checklist:
